03/Task03.js

```diff
@@ -1,10 +1,43 @@
 import React from 'react';
+import {Route, useParams, Redirect} from 'react-router-dom';
 
+import Shop from "../src/components/Shop";
 import products from './../src/products.json';
 
 const Task03 = () => {
+    const DisplayShop = () => {
+        const { cat='' } = useParams()
+
+        const productList = products.filter(product => {
+            product.link = `/task02/product-${product.id}`
```

Comment:
W tym zadaniu te linki nie działają - jak myślisz dlaczego? Postaraj się to poprawić ;)